### PR TITLE
Update CHANGELOG.rst

### PR DESCRIPTION
**Pull request by YueErro**
Feel free to change the file.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,10 +1,30 @@
...
+
+* Whole infraestructure base on HRIM Coliza
+* Compilance with https://github.com/AcutronicRobotics/robotiq_modular_gripper/tree/crystal
+* Models for Reinforcement Learning
+* Two MARA robots
+* RViz2 visualization
+
+**Changed**
+
+* Measurement values in the URDFs
```
Comment (rkojcev) [inline CHANGELOG.rst]: I would say something mechanical properties of each link in the urdf.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
+* Instructions in readme
+* Installation of additional ROS2 dependencies trough Debian packages, instead of from source
+* Topic names
+* Files (unification)
+
+**Removed**
```
Comment (rkojcev) [inline CHANGELOG.rst]: can we add list here of things we removed. We had an issue regarding that

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,10 +1,30 @@
...
+
+* Unused/Deprecated files
```
Comment (rkojcev) [inline CHANGELOG.rst]: also here if there are any files we no longer use just to inform users of this repo

**Comment by YueErro** [comment]
Feel free to change file.

**Comment by rkojcev** [comment]
Lets do this better. I will generate log files using a tool. @YueErro please close this

**Pull request closed without merging**